gef-ui/server/routes/unissued-facilities.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const { applicationDetails } = require('../controllers/application-details');
+const {
+  changeUnissuedAboutFacility,
+  changeUnissuedAboutFacilityChange,
+  postChangeUnissuedAboutFacility,
+  postChangeUnissuedAboutFacilityChange,
+} = require('../controllers/unissued-facilities');
+const validateToken = require('../middleware/validateToken');
+
+const router = express.Router();
+
+// unissued facility list
+router.get('/application-details/:dealId/unissued-facilities', validateToken, (req, res) => applicationDetails(req, res));
+// get change unissued facility from facility list
+router.get('/application-details/:dealId/unissued-facilities/:facilityId/about-facility', validateToken, (req, res) =>
+  changeUnissuedAboutFacility(req, res));
+// get change unissued facility from application preview
+router.get('/application-details/:dealId/unissued-facilities-change/:facilityId/about-facility', validateToken, (req, res) =>
+  changeUnissuedAboutFacilityChange(req, res));
+// post change unissued facility to issued from facility list
+router.post('/application-details/:dealId/unissued-facilities/:facilityId/about-facility', validateToken, (req, res) =>
+  postChangeUnissuedAboutFacility(req, res));
+// post change unissued facilities to issued from application preview
+router.post('/application-details/:dealId/unissued-facilities-change/:facilityId/about-facility', validateToken, (req, res) =>
+  postChangeUnissuedAboutFacilityChange(req, res));
+
```

Comment:
I'm not familiar with the design/page etc - finding some of these controller namings confusing, `changeUnissuedAboutFacilityChange`, is there a better/simpler name we could use?

Either way would be good to add an empty line in between each route here so it's easier to read

Reply:
done